docs/software-platforms/aws/working-with-s3-buckets.md

```diff
@@ -40,4 +40,36 @@ You can then use `$OPENSCPCA_RESULTS_BUCKET` (or, `${OPENSCPCA_RESULTS_BUCKET}`)
 
 ## Syncing your results to S3
 
-_Coming up in the next PR._
+We have written a script to help you sync you results to your S3 bucket, stored in [`scripts/sync-results.py`](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/scripts/sync-results.py).
+You will need to use the [terminal](../general-tools/using-the-terminal.md) to run this script.
```

Comment:
Are we saying this for all scripts? I am not sure we need the explicit call out here, unless we have another way for people to run scripts.

Reply:
No, we don't explicitly say this every time. I can remove it.